.vib/mediawiki/goss/mediawiki.yaml

```diff
@@ -0,0 +1,44 @@
+group:
+  daemon:
+    exists: true
+user:
+  daemon:
+    exists: true
+file:
+  /opt/bitnami/mediawiki:
+    exists: true
+    filetype: directory
+    mode: "0775"
+    owner: daemon
+    group: root
+  /bitnami/mediawiki:
+    exists: true
+    filetype: directory
+    mode: "0775"
+    owner: daemon
+    group: root
+  /opt/bitnami/mediawiki/images:
+    exists: true
+    filetype: directory
+    mode: "0775"
+    owner: daemon
+    group: root
+  /opt/bitnami/mediawiki/cache:
+    exists: true
+    filetype: directory
+    mode: "0775"
+    owner: daemon
+    group: root
+  /opt/bitnami/php/etc/php.ini:
+    exists: true
+    filetype: file
+    contains:
+      - "memory_limit = 256M"
+  /opt/bitnami/mediawiki/HISTORY:
+    exists: true
+    filetype: file
+    contains:
+      - "RewriteEngine On"
+      - "RewriteRule"
```

Comment:
I'm sorry but I can't find where this test comes from, could you please provide me with some more details?

I see this in the `postunpack.sh`:
```
ensure_web_server_app_configuration_exists "mediawiki" --type php --apache-extra-directory-configuration "
RewriteEngine On
RewriteRule ^/?wiki(/.*)?$ %{DOCUMENT_ROOT}/index.php [L]
"
```
But the affected file is not `/opt/bitnami/mediawiki/HISTORY` but `/opt/bitnami/apache/conf/vhosts/mediawiki-vhost.conf`:

```
$ cat /opt/bitnami/apache/conf/vhosts/mediawiki-vhost.conf 
<VirtualHost 127.0.0.1:8080 _default_:8080>
  ServerName www.example.com
  ServerAlias *
  DocumentRoot /opt/bitnami/mediawiki
  <Directory "/opt/bitnami/mediawiki">
    Options -Indexes +FollowSymLinks -MultiViews
    AllowOverride None
    Require all granted
    RewriteEngine On
    RewriteRule ^/?wiki(/.*)?$ %{DOCUMENT_ROOT}/index.php [L]
  </Directory>
  Include "/opt/bitnami/apache/conf/vhosts/htaccess/mediawiki-htaccess.conf"
</VirtualHost>
```